### PR TITLE
confirmation export letter tests added

### PR DESCRIPTION
**Pull request by timstone**
Added feature to test back office confirmation export

### CODE REVIEW

**Comment by Cruikshanks** [comment]
This is ok to merge @timstone Hakiri is having a bit of a dickie at the mo, but I can the tests have run and they have passed.

File: features/step_definitions/back_office/general_steps.rb [hunk 1]
```diff
@@ -197,6 +197,30 @@
...
+When(/^I export confirmation letters for today$/) do
+  @app.search_page.nav_bar.registrations_menu.click
+  @app.search_page.nav_bar.confirmation_letters.click
+
+  # finds today's date and saves them for use in export from date
+  # 'To' date defaults to today's date is isn't required
+  time = Time.new
+
+  @year = time.year
+  @month = time.strftime("%B")
```
Comment (Cruikshanks) [inline features/step_definitions/back_office/general_steps.rb:263]: Any reason why `time.month` doesn't work here @timstone ?

**Review by Cruikshanks — COMMENTED** [review]
Just that one query @timstone . Other than that its good to go.

File: features/step_definitions/back_office/general_steps.rb (hunk 1 above)
Comment (timstone) [reply features/step_definitions/back_office/general_steps.rb:263]: That returns a number, I wanted the month's name.

**Pull request merged**